src/core/RestValue.ts

```diff
@@ -35,18 +35,18 @@ export class RestValue implements IdentifiedDataSerializable {
      * HTTP Content-Type specified for the value.
      */
     contentType: string;
-    /** @internal */
+    /** @ignore */
```

Comment:
No. The change is done because of `RestValue` not being exported in the `index.ts` previously. Once I have fixed that, `npm run validate-user-code` started failing. So, user code validation has already proven itself as a valuable test.